lua/GPM.lua

```diff
@@ -706,8 +698,8 @@ function GPM(data)
 	verifyUnnecessaryArguments(data, {"origin", "distance", "progress", "destination", "strategy"})
 	mandatoryTableArgument(data, "origin", "CellularSpace")
 
-	if not data.origin.geometry then
-		customError("The CellularSpace in argument 'origin' must be loaded with 'geometry = true'.")
+	if data.origin.geometry == false then
+		customError("The CellularSpace in argument 'origin' must be loaded without 'geometry'.")
```

Comment:
The message should be the original one. It must be loaded WITH geometry. The only difference is that now the alternative test show the error when the user explicitly sets `geometry = false`.

Reply:
But now, the default is not 'true'?
If user put 'geometry = true', he will receiver a error.

Reply:
You are confusing base package with gpm package. If the user sets `geometry = false` and use the `CellularSpace` to create a GPM/Network, it will stop with an error saying that the user should use `geometry = true`. It does not matter if this is the default value. If he sets true manually, she will get a warning saying that this argument could be removed, but this is in charge of base package, not gpm.

Reply:
There is no set 'geometry' for gpm, only for cellspace. If user set 'geometry = true' this happens:
Error: Argument 'geometry' could be removed as it is the default value (true).

Reply:
`"without using argument 'geometry'"`